tests/python/integration/test_integration.py

```diff
@@ -5,36 +5,39 @@
 
 """Each scenario tests the three output formats"""
 
-def add_test_scenario(user_dss_clients,scenario_id):
+
+def add_test_scenario(user_dss_clients, scenario_id):
```

Comment:
This is a leftover for a dummy test?

Reply:
No this is the generic method called in each test below to run the integration test

Reply:
ah yes, my bad